sw/tools/STDMA_dongle/README.md

```diff
@@ -0,0 +1,37 @@
+STDMA_dongle.md
+
+### Description
+This tool allows Bluegiga dongle to communicate using STDMA (Self-organized Time-Division Multiple Access).
+
+Using this, drones with a Bluegiga USB Bluetooth dongle can communicate with eachother and exchange their RSSI (i.e., signal strenth, which is a measure of distance). This has been tested with up to 3 ARDrones.
+
+It was used in: Coppola, M., McGuire, K.N., Scheper, K.Y.W. et al. Auton Robot (2018). https://doi.org/10.1007/s10514-018-9760-3
+
+### Commands functions
+make clean && make --> makes it for the computer
+EMBED=1 --> this builds it for the drone, rather than for your computer
+Once connected to the drone --> make upload_program EMBED=1 HOST=192..........
+
+### Setting up the communication between laptops/drones
+Turn on the drones and go to:
+
+	cd $PAPARAZZI_HOME/sw/tools/STDMA_dongle
+
+The command
```

Comment:
```suggestion

Select your **DRONE** type in the Makefile (tested with ARDrone)

The command
```

Reply:
why not also adding `DRONE ?= ...` so that we can change the type of drone without necessarily changing the file itself ?

Reply:
hum, I wanted to make a modification proposal but apparently it directly applies it. @dewagter what is the trick to make a commit suggestion ?

Reply:
The modification proposal (insert a suggestion) is available when you make a comment about a line of code and hit Ctrl-G or press the +- icon and change the code within.